app/src/main/java/com/eis/audioUtility/AudioUtilityManager.java

```diff
@@ -0,0 +1,220 @@
+package com.eis.audioUtility;
+
+import android.content.Context;
+import android.media.AudioManager;
+import android.os.Build;
+import android.os.VibrationEffect;
+import android.os.Vibrator;
+
+import androidx.annotation.NonNull;
+
+import static android.content.Context.AUDIO_SERVICE;
+
+/**
+ * This is an Utility class used to manage main Audio Streams (Alarm, Music, and Ring),
+ * and to manage Vibration
+ *
+ * @author Francesco Bau'
+ */
+public class AudioUtilityManager {
+
+    public static final int MAX_PERCENTAGE = 100;
+    public static final int MIN_PERCENTAGE = 0;
+
+    private static final int VIBRATION_TIME = 500;
+    private static final int VIBRATION_DELAY = 500;
+    private static final int VIBRATION_REPEAT_CODE = 1;
+
+    // Needed to don't show any flag from any Volume variation.
+    private static final int ZERO_FLAG_CODE = 0;
+
+    public enum AUMStream {
+        ALARM,
+        RING,
+        MUSIC
+    }
+
+    /**
+     * @param context The current Context
```

Comment:
I get that the name is obvious etc. but I would have added some more documentation anyway. Something like
`Returns the current AudioManager instance`

Reply:
Thank you, I've fixed it.